backlog/src/Program.cs

```diff
@@ -19,9 +19,22 @@ public static int Main(string[] args)
         {
             try
             {
-                if (args.Length < 2 || args[0] != "--id" || !uint.TryParse(args[1], out uint id))
+                uint? id = null;
```

Comment:
If this gets any more complicated, this should probably be a function. I think it struggles because the logic is inverted (pass through if there's no args or there's `--id` and the other arg is OK, but it's written stop if the opposite)